src/Registration/cReg/NiftiImageData.cpp

```diff
@@ -222,6 +226,12 @@ NiftiImageData<dataType> NiftiImageData<dataType>::operator*(const float& val) c
     return maths(val,mul);
 }
 
+template<class dataType>
+NiftiImageData<dataType> NiftiImageData<dataType>::operator/(const float& val) const
```

Comment:
```suggestion
NiftiImageData<dataType> NiftiImageData<dataType>::operator/(const float val) const
```
generally never use a ref for `const float/int` etc. it's unclear and would just slow it down.

Reply:
I see it's done with `*` as well though. Maybe accept this one, and then do a separate PR for all of those (not urgent)